masq/src/commands/change_password_command.rs

```diff
@@ -71,43 +86,68 @@ impl Command for ChangePasswordCommand {
         };
         let _: UiChangePasswordResponse =
             transaction(input, context, STANDARD_COMMAND_TIMEOUT_MILLIS)?;
-        short_writeln!(context.stdout(), "Database password has been changed");
+        short_writeln!(context.stdout(), "{}", CHANGE_PASSWORD_MESSAGE);
         Ok(())
     }
 
     as_any_impl!();
 }
 
+const CHANGE_PASSWORD_COMMAND: &str = "change-password";
+const CHANGE_PASSWORD_ABOUT_MESSAGE: &str = "Changes the existing password on the Node database";
+
+const OLD_DB_PASSWORD_NAME: &str = "old-db-password";
+const OLD_DB_PASSWORD_HELP: &str = "The existing password";
+const OLD_DB_PASSWORD_VALUE_NAME: &str = "OLD-DB-PASSWORD";
+const OLD_DB_PASSWORD_INDEX: u64 = 1;
+const OLD_DB_PASSWORD_REQUIRED: bool = true;
+const OLD_DB_PASSWORD_CASE_INSENSITIVE: bool = false;
+
+const NEW_DB_PASSWORD_NAME: &str = "new-db-password";
+const NEW_DB_PASSWORD_HELP: &str = "The new password to set";
+const NEW_DB_PASSWORD_VALUE_NAME: &str = "NEW-DB-PASSWORD";
+const NEW_DB_PASSWORD_INDEX: u64 = 2;
+const NEW_DB_PASSWORD_REQUIRED: bool = true;
+const NEW_DB_PASSWORD_CASE_INSENSITIVE: bool = false;
```

Comment:
Here and also at other places where you tested flags at Clap subcommands you can preserve it, there is some value in that, at least I think...so go ahead if you want :)

Reply:
The number of arguments inside these Clap Subcommands  will drastically increase in other files. For Example, if we have 5 arguments in 5 subcommands, it'll require creating 25 new constants. I already did in other files, and it started looking clumsy.

What I may recommend using is Static HashMaps using the external crate Rust-phf.

As mentioned in this answer - https://stackoverflow.com/questions/60273064/rust-best-practices-when-specifying-a-constant-hash-map